bin/docker-deploy.sh

```diff
@@ -57,12 +57,19 @@ return 0
 
 start_deploy(){
  KUBE_OPTS=""
- 
+
  if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
    KUBE_OPTS="$KUBE_OPTS --container=$app_name"
  fi
 
- kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"	
+ kubectl rolling-update "$app_name" "--image=$repo:$tag" $KUBE_OPTS "--server=$KUBE_MASTER" "--namespace=$username" "--image-pull-policy=IfNotPresent"
+
+ if [ "$MULTIPLE_CONTAINER_PODS" == "true" ]; then
```

Comment:
can you avoid calling kubectl in this if check?

Basically,
if(multiple_container_pods == true)
  TO_DEPLOY= $DEPLOYABLE_CONTAINERS
else
  TO_DEPLOY="main"

Reply:
👍